.rubocop.yml

```diff
@@ -21,6 +21,7 @@ AllCops:
     - 'config/environments/*.rb'
     - 'db/**/*'
     - 'vendor/**/*'
+    - 'tmp/**/*'
```

Comment:
if you upload files that look like code to a test object in localhost, this get thrown into a subfolder under tmp... we do not want rubocop running on these files